documentation/resources/filters/stanza.md

```diff
@@ -0,0 +1,28 @@
+# Examples for the stanza filter
+
+## Original file
+````
+[first]
+line1 = value1
```

Comment:
to match the formatting, should all lines under the bracketed headings be indented? line 8, 15, & 18 also

Reply:
I'll make it prettier and line them up.  The inserted lines won't necessarily match the existing format.